lms/static/sass/bootstrap/_layouts.scss

```diff
@@ -1,49 +1,81 @@
 // LMS layouts
 
 .content-wrapper {
-    margin-top: $baseline;
-
-    .course-tabs {
-        padding-bottom: 0;
-
-        .nav-item {
-            &.active, &:hover{
-                .nav-link {
-                    border-bottom-color: theme-color("primary");
-                    color: theme-color("primary");
-                }
-            }
-
-            .nav-link {
-                padding: $baseline/2 $baseline*3/4 $baseline*13/20;
-                border-style: solid;
-                border-width: 0 0 $baseline/5 0;
-                border-bottom-color: transparent;
-
-                @include media-breakpoint-down(md) {
-                    border: none;
-                    text-align: left;
-                    padding: 0 0 $baseline/2 0;
-                }
-            }
-        }
-    }
+  margin-top: $baseline;
+  padding-bottom: $baseline*2;
 
-    .main-container {
-        border: 1px solid $border-color;
-        background-color: $body-bg;
+  .course-tabs {
+    padding: 0;
+    font-size: $font-size-sm;
 
-        .page-header {
-            border-bottom: 1px solid $border-color;
-            padding: 20px;
+    .nav-item {
+      &.active,
+      &:hover {
+        .nav-link {
+          border-bottom-color: theme-color("primary");
+          color: theme-color("primary");
         }
+      }
+
+      .nav-link {
+        padding: $baseline/2 $baseline*3/4 $baseline*13/20;
+        border-style: solid;
+        border-width: 0 0 $baseline/5 0;
+        border-bottom-color: transparent;
 
-        .page-content {
-            padding: 20px;
+        @include media-breakpoint-down(md) {
```

Comment:
I was debating that. There are some instances that have a lot of tabs so this might end up wrapping before the breakpoint. I'll try the smaller value and see how it looks.